app/src/main/java/edu/wpi/first/shuffleboard/app/components/WidgetPane.java

```diff
@@ -125,7 +129,8 @@ private void resizeTiles() {
    * Gets the tile for the widget containing the given source.
    */
   public Optional<WidgetTile> widgetForSource(DataSource<?> source) {
```

Comment:
I think this should return all tiles, not just widget tiles, that contain a widget for the source.

Reply:
unused because it's no longer used. you might have been thinking of dashboard.selectWidgets ?